### DISCUSSION TITLE
DIA-SWATH metabolomics data pre-treatment using Mzmine 3 - recommended procedure

### DISCUSSION THREAD

**Discussion by ThmsBrnt** [mzmine/mzmine#1095] [Q&A]
Hi,

We are trying to process DIA-SWATH data from Sciex QTOF with Mzmine 3 and we are having some trouble understanding the steps. Would you have recommendations about the procedure to apply for DIA data pretreatment?
Particularly, how to deconvolute the MS/MS spectra to assign them to the correct m/z precursor?
Thank you for your help.

Thomas Brunet.

**Comment by robinschmid** [comment]
Currently MZmine does not really support SWATH. We are still searching for people who use it and are willing to contribute the modules to stitch together SWATH data.

**Comment by ThmsBrnt** [comment]
Hi,
We are currently working on the application of DIA-SWATH to metabolomics and lipidomics on a SCIEX ZenoTOF 7600 system.
At the moment we are using MSDIAL to process SWATH data but we would be interested in collaborating with the Mzmine team on the DIA software implementation.

Regards,
----------------------------------------
Thomas BRUNET
PhD student in analytical chemistry
Institut des Sciences Analytiques (UMR 5280) - Équipe "Biological Analysis by Mass Spectrometry"
5 rue de la Doua, 69100 Villeurbanne, FRANCE
----------------------------------------
+33 (0)4 37 42 36 80
e-mail : ***@***.******@***.***>

De : Robin Schmid ***@***.***>
Envoyé : 20 February 2023 12:17
À : mzmine/mzmine3 ***@***.***>
Cc : BRUNET THOMAS p2008910 ***@***.***>; Author ***@***.***>
Objet : Re: [mzmine/mzmine3] DIA-SWATH metabolomics data pre-treatment using Mzmine 3 - recommended procedure (Discussion #1095)


Currently MZmine does not really support SWATH. We are still searching for people who use it and are willing to contribute the modules to stitch together SWATH data.
